### PR TITLE
Add pt-br.json

### PR DESCRIPTION
**Pull request by pstwh**
Brazilian Portuguese JSON language file.

# Description

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Closes: # (issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added a short summary of my change to the CHANGELOG.md
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by freddyaboulton** [comment]
Thank you @pstwh ! Can you please update the CHANGELOG?

File: ui/packages/app/src/lang/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (freddyaboulton) [inline ui/packages/app/src/lang/pt-BR.json:1]: I think we need to rename to `pt-br.json` for the test to pass!

File: CHANGELOG.md [hunk 2]
```diff
@@ -104,6 +104,7 @@ No changes to highlight.
 * Make try examples button more prominent by [@aliabd](https://github.com/aliabd) in [PR 2705](https://github.com/gradio-app/gradio/pull/2705)
 * Fix id clashes in docs by [@aliabd](https://github.com/aliabd) in [PR 2713](https://github.com/gradio-app/gradio/pull/2713)
 * Fix typos in guide docs by [@andridns](https://github.com/andridns) in [PR 2722](https://github.com/gradio-app/gradio/pull/2722)
+* Add pt-br.json by [@pstwh](http://github.com/pstwh) in [PR 2753](https://github.com/gradio-app/gradio/pull/2753)
```
Comment (abidlabs) [inline CHANGELOG.md:107]: This should be higher up in the changelog, in the "Upcoming Release" section, not in the "3.12.0" section

Comment (pstwh) [reply CHANGELOG.md]: Fixed, sorry for my inattention.

Comment (abidlabs) [reply CHANGELOG.md]: No worries, thanks for the fix!

File: CHANGELOG.md [hunk 3]
```diff
@@ -54,10 +54,11 @@ No changes to highlight.
 No changes to highlight.
 
 ## Full Changelog:
+* Images in the chatbot component are now resized if they exceed a max width by [@abidlabs](https://github.com/abidlabs) in [PR 2748](https://github.com/gradio-app/gradio/pull/2748)
+* Add Brazilian Portuguese translation (pt-br.json) by [@pstwh](http://github.com/pstwh) in [PR 2753](https://github.com/gradio-app/gradio/pull/2753)
```
Comment (freddyaboulton) [inline CHANGELOG.md:58]: Final nit: Can you add an image here showing an example gradio demo translated to Portuguese! Would be really nice visual. If you edit the file directly on github, you just can drag and drop an image.

Thank you!

**Comment by abidlabs** [comment]
I'll take the liberty to add the image to the changelog so that we can merge this in. Thanks @pstwh for the translations!

**Review by abidlabs — APPROVED** [review]

**Pull request merged**